### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by dyinglights** [TPPIDev/TestPackPleaseIgnore2#296]
I have been playing for a while and suddenly have gotten this crash on startup. No idea why.

Linky ~ http://paste.ubuntu.com/10883576/

thanks

**Comment by TehNut** [comment]
Is it reproducable?

**Comment by dyinglights** [comment]
Yeah sometimes I can get to loading the textures of the world, other times not that far.
Here is a log from when I just ran it.
~ http://paste.ubuntu.com/10883767/

**Comment by dyinglights** [comment]
Heyo I didn't see this email.
It happens 100% of the time, but after varying length of time waiting for the world to load. A new world works fine.
I zipped and attached all the logs in case it it more than one error.Cheers, Rob
Date: Fri, 24 Apr 2015 22:25:12 -0700
